mirdata/annotations.py

```diff
@@ -79,6 +79,26 @@ def __repr__(self):
         return repr_str
 
 
+class MultiAnnotator(Annotation):
+    """Multiple annotator class.
+    This class should be used for datasets with multiple annotators (e.g. multiple annotators per track).
+
+    Attributes:
+        annotators (list): list with annotator ids
+        annotations (list): list of annotations (e.g. [annotations.BeatData, annotations.ChordData]
+    """
+
+    def __init__(self, annotators, annotations) -> None:
+        validate_array_like(annotators, list, str, none_allowed=True)
+        validate_array_like(
+            annotations, list, Annotation, check_child=True, none_allowed=True
```

Comment:
should the validation also check that the `Annotation` subtypes are all the same?